seqr/utils/search/utils.py

```diff
@@ -426,8 +426,10 @@ def _validate_sort(sort, families):
 def _search_dataset_type(search):
     locus = search['parsed_locus']
     parsed_variant_ids = locus.get('parsed_variant_ids', locus['variant_ids'])
-    if parsed_variant_ids:
-        return Sample.DATASET_TYPE_VARIANT_CALLS, None, _variant_ids_dataset_type(parsed_variant_ids)
+    rsids = locus.get('rs_ids')
```

Comment:
there were a couple edge cases where the shared utility for determining which dataset types were applicable to search were unnecessarily broad. Rather than adding logic in clickhouse search to handle this (i.e. make RSID search work properly against SVs) I instead opted to update the shared logic to prevent these searches from running in the first place. While this does impact other backends, its an overall improvement and keeps the clickhouse logic cleaner, so I think it is worth it